tests/tracing/test_fluent.py

```diff
@@ -370,12 +370,14 @@ def predict(self, x, y):
     TRACE_BUFFER.clear()
 
     # Exception during inspecting inputs: trace is logged without inputs field
-    with mock.patch("mlflow.tracing.utils.inspect.signature", side_effect=ValueError("Some error")):
+    with mock.patch(
+        "mlflow.tracing.fluent.capture_function_input_args", side_effect=ValueError("Some error")
+    ):
         output = model.predict(2, 5)
```

Comment:
```suggestion
    with mock.patch(
        "mlflow.tracing.fluent.capture_function_input_args", side_effect=ValueError("Some error")
    ) as mock_input_args:
        output = model.predict(2, 5)
        mock_input_args.assert_called_once()
```

can we use `assert_called_once` to make sure `capture_function_input_args` is really called?

Reply:
Great idea!